app/main.py

```diff
@@ -1,9 +1,18 @@
+from app.routes import user, chat
 from fastapi import FastAPI
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from app.models import Base
 
 app = FastAPI()
 
-# Import the routers
-from app.routes import user, chat
+# Configure the database connection
+DATABASE_URL = "sqlite:///./app.db"
```

Comment:
I think is better idea to set this value on settings.py and bring it from there (Just imagine how tedious would it be to work with this having several environments like testing, stagging and production)